subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/executing/TestExecutor.kt

```diff
@@ -49,6 +49,8 @@ interface TestExecutor {
         ) {
             withDevices(reservations = reservations(testsToRun)) { devices ->
 
+                logger.info("Got devices channel")
```

Comment:
```suggestion
                logger.debug("Got devices channel")
```

Reply:
removed it completely, not necessary message